src/util/codec/number.rs

```diff
@@ -11,7 +11,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use byteorder::{BigEndian, LittleEndian, ReadBytesExt, WriteBytesExt};
+// FIXME(shirly): remove following later
```

Comment:
Can this be removed now?

Reply:
no, it will be removed after all Read trait in `datum`  been removed.